### ISSUE TITLE
Support TMS Vector Tiles

### ISSUE THREAD

**Issue by LHBruneton-C2C** [camptocamp/inkmap#64]
Hello!
Reading the api and the source code, it seems that TMS Vector Tiles are not supported by InkMap.
Is that a correct analysis? Is there any work planned for this feature?
Thanks in advance for you answer.

**Comment by jahow** [comment]
Indeed TMS is not explicitly supported, but maybe declaring it as a XYZ later with the correct url template could be enough? Something that might be useful is also adding a bounding box option to tiled layers to avoid querying tiles outside of the available extent.

**Comment by LHBruneton-C2C** [comment]
About XYZ layers, aren't those explicitly for raster images?
I will keep the bounding box option in mind, thanks!

**Comment by jahow** [comment]
Yes an XYZ layer is simply a layer of tiled images following a URL template, which a TMS layer also is I think. I'm not sure if that would work out for other projections than web mercator though.

**Comment by LHBruneton-C2C** [comment]
OK, thanks!
Our need would be to support this server: https://data.geopf.fr/tms/1.0.0
Further documentation here: https://geoservices.ign.fr/documentation/services/services-geoplateforme/diffusion#70064
As you can see, these will be vector tiles and not raster. Even if it follows a TMS scheme.

**Comment by jahow** [comment]
Implemented in #76
